index.html

```diff
@@ -76,6 +76,13 @@
 	$generatedURL = $('#generated-url');
 $form.submit(function(e){
 	e.preventDefault();
+	window.open($generatedURL.attr('href'), '_blank');

```

Comment:
I don't think it will be blocked because the submit event is initiated by user. So far i tested on a few browsers that I have it's okay :D